_posts/2023-03-22-cockpit-288.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Cockpit 288
+author: kkoukiou
+date: '2023-03-22'
+tags: cockpit, machines, podman
+slug: cockpit-288
+category: release
+summary: 'Metrics redesign, accounts page improvements, virtual machines cloud base editing, services page speed ups, chronyd supports timeservers, and esbuild bundler'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 288, cockpit-machines 286, and cockpit-podman 65:
+
+## Accounts: Show shell and home directory on an account's detail page
+
+Account detail pages show the home directory and shell for an account.
+
+![details page for an account, showing the newly added shell and home directory entries]({{ site.baseurl }}/images/288-accounts-show-shell-and-home-directory.png)
+
+## Accounts: Custom user ID during account creation
+
+Custom user ID can now be specified during account creation. Initially, it's prefilled with the next available user ID on the system.
+
+![account creation dialog showing an entry for user ID]({{ site.baseurl }}/images/288-accounts-custom-user-id-during-account-creation.png)
+
+## Overview: Support additional timeservers with chronyd
+
+Cockpit can now configure additional time servers when chronyd is used to synchronize the clock. Previously, this was only supported with systemd-timesyncd.
+
+![screenshot of support for additional timeservers with chronyd]({{ site.baseurl }}/images/288-overview-support-for-additional-timeservers-with-chronyd.png)
+
+## Metrics: Redesigned to show more information by default
+
+The PCP metrics graphs have been redesigned so as to show by default more data in a single page view.
```

Comment:
```suggestion
The PCP metrics graphs have been redesigned to only show a summary of events for each hour, so that more data fits on the page. The hours can be expanded to get more details.
```